pkg/sink/codec/common/config.go

```diff
@@ -177,6 +181,8 @@ type urlConfig struct {
 	// EncodingFormatType is only works for the simple protocol,
 	// can be `json` and `avro`, default to `json`.
 	EncodingFormatType *string `form:"encoding-format"`
+
+	OutputRowKey *bool `form:"output-row-key"`
```

Comment:
please add a comment to show what the item means, and it only works for canal-json now.